lib/kubeclient/common.rb

```diff
@@ -24,6 +24,10 @@ def initialize_client(
         password: nil,
         bearer_token: nil,
         bearer_token_file: nil
+      },
+      socket_options: {
```

Comment:
Travis is failing on a Rubocop check, I mentioned it here https://github.com/abonas/kubeclient/pull/142#issuecomment-161314282